sbin/Makefile.am

```diff
@@ -55,12 +28,29 @@ if ENABLE_VNETBUILD
 scripts += vnetbuild
 endif
 
-sbin_SCRIPTS = $(scripts)
+CLEANFILES = install.config
 
-EXTRA_DIST = \
-	commands.sed.in \
-	$(scriptsin) \
-	$(inclib_DATA)
+inclib_DATA = \
+	functions.common \
+	install.config \
+	$(NULL)
+
+inclib_SCRIPTS = $(scripts)
 
-uninstall-local:
+EXTRA_DIST = \
+	functions.common \
+	install.config.in \
+	$(scripts) \
+	$(NULL)
+
+install-exec-hook:
+	$(MKDIR_P) $(DESTDIR)$(sbindir)
```

Comment:
you should probably also use install or chmod as if umask is incorrect you get unreadable files.